internal/pkg/cli/dependencies/container.go

```diff
@@ -143,6 +143,11 @@ func (c *Container) ProjectDir() (filesystem.Fs, error) {
 			return nil, ErrProjectManifestNotFound
 		}
 
+		// Check version field
+		if err := version.CheckLocalVersion(c.Logger(), c.fs, projectManifest.Path()); err != nil {
```

Comment:
Tu som presunul kontrolu toho, ci je manifest aktualnej verzie.
Kazdy prikaz, ktory potrebuje `ProjectDir` to automaticky skontroluje.